ktor-server/ktor-server-plugins/ktor-server-metrics/jvm/src/io/ktor/server/metrics/dropwizard/DropwizardMetricsUtils.kt

```diff
@@ -5,23 +5,10 @@
 package io.ktor.server.metrics.dropwizard
 
 import com.codahale.metrics.*
-import io.ktor.server.application.*
 import io.ktor.util.*
 
 internal class RoutingMetrics(val name: String, val context: Timer.Context)
 internal val routingMetricsKey = AttributeKey<RoutingMetrics>("metrics")
```

Comment:
One thing we noticed is that name of this 2 keys are the same. 

Because of this looking up in the attributes map fails with the class cast exception in some cases.

Reply:
yes, fixed